src/azure-cli/azure/cli/command_modules/acr/task.py

```diff
@@ -982,3 +989,11 @@ def _get_trigger_event_list_patch(cmd,
             if SourceTriggerEvent.pullrequest.value in source_trigger_events:
                 source_trigger_events.remove(SourceTriggerEvent.pullrequest.value)
     return source_trigger_events
+
+
+def _get_branch_name(context_path):
```

Comment:
We should default to `master` branch here if not found.

Reply:
I'm not doing this because in update the current logic is to send none by default. Should we switch to master when the branch is not specified in a context when updating a task?

Reply:
No, you can leave it as None.